osmaxx/excerptexport/_settings.py

```diff
@@ -1,8 +1,17 @@
 import os
+from datetime import timedelta
 
 from django.conf import settings
 
 POLYFILE_LOCATION = os.path.abspath(os.path.join(os.path.dirname(__file__), 'data', 'polyfiles'))
+OLD_RESULT_FILES_REMOVAL_CHECK_INTERVAL_HOURS = timedelta(hours=1)  # default every hour
+PURGE_OLD_RESULT_FILES_AFTER = timedelta(14)  # default to two weeks
```

Comment:
Consider passing the days as key word argument: `timedelta(days=14)`

This would remove any doubt or confusion about what time unit is used.